TLM/TLM/UI/MainMenu/TimedTrafficLightsButton.cs

```diff
@@ -1,12 +1,12 @@
-﻿namespace TrafficManager.UI.MainMenu {
+namespace TrafficManager.UI.MainMenu {
     using State;
 
     public class TimedTrafficLightsButton : MenuToolModeButton {
         protected override ToolMode ToolMode => ToolMode.TimedLightsSelectNode;
 
         protected override ButtonFunction Function => ButtonFunction.TimedTrafficLights;
 
-        public override string Tooltip => Translation.Menu.Get("Tooltip:Timed traffic lights");
+        public override string Tooltip => Translation.Menu.Get("Tooltip:Timed traffic lights") + "\n" + Translation.Menu.Get("Tooltip.Keybinds:Timed traffic lights");
```

Comment:
We don't have that language key yet 😕

Reply:
Don't know how it got here. I got confused. This belongs to another review.